src/tag/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from utils.models import DefaultModel
+
+# We ask openalex to classify a paper into concepts, which we then present as tag candidates from which users can pick zero or more to attach to the paper.
+# TODO: migration
+# https://docs.openalex.org/about-the-data/concept
+class Concept(DefaultModel):
```

Comment:
Instead of calling this Concept (kind of generic), can we name this to something else? Like OpenAlexConcept